baseband/vdif/header.py

```diff
@@ -357,7 +360,7 @@ def station(self, station):
         assert int(station_id) == station_id
         self['station_id'] = station_id
 
-    def get_time(self, sample_rate=None, frame_nr=None):
+    def get_time(self, sample_rate=None):
```

Comment:
'ref_epoch' and plain ref_epoch -> \`ref_epoch\`; 'seconds' and plain second -> \`seconds\`, frame_nr -> \`frame_nr\`, etc.

Reply:
Actually, inconsistently using backquotes for header properties is an issue with all Mark 5B and VDIF docs for `get_time` and `set_time`.  Either always use backquotes or never use them, and I think always using them is more consistent with our other docstrings.

Reply:
OK, but let's do that as part of another PR.